src/svelte/Creature.svelte

```diff
@@ -27,25 +27,32 @@
         setIcon(div, "eye-off");
     };
 
+    var hoverTimeout = null;
```

Comment:
Why set this here (and why with var?)

Reply:
var or let ... Just a relapse. Can fix that. This pushes opening of the hover/popover until the timeout occurs.. which means you need to cancel the timeout if you don't want the popover to occur (i.e. the mouse left before it popped)...